angel_system/activity_classification/tcn_hpl/predict.py

```diff
@@ -112,6 +112,17 @@ def normalize_detection_features(
     if that is not desired.
 
     :param det_feats: Object Detection features to be normalized.
+    :param feature_version: Version of the feature conversion approach.
+    :param top_k_objects: Number top confidence objects to use per label, defaults to 1
+    :param image_width: Integer pixel width of the image that object detections
+        were generated on.
+    :param image_height: Integer pixel height of the image that object
+        detections were generated on.
+    :param num_det_classes: Number of object detection classes (note: DOES include the ahnd labels but DOES NOT include the patient and user labels)
```

Comment:
```suggestion
    :param num_det_classes: Number of object detection classes (note: DOES include the hand labels but DOES NOT include the patient and user labels)
```

Reply:
@cameron-a-johnson question for the `deactivate_thresh_frame_count` value - but I don't think we'll really know until we have good models to test with the other tasks